.github/workflows/publish_release.yml

```diff
@@ -124,11 +124,17 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           branch: "1.x"
 
+      - name: Prepare changelog release notes
+        id: changelog-notes
+        run: |
+          notes="$(echo "${{ steps.notes.outputs.release-notes }}" | sed "s/^## /### /")"
+          echo "release-notes=$notes" >> $GITHUB_OUTPUT
+
```

Comment:
The [changelog](https://github.com/laravel/nightwatch/blob/1.x/CHANGELOG.md) expects the heading to be `h3`. The GitHub release will continue to use `h2`